modules/ROOT/pages/mule-maven-plugin.adoc

```diff
@@ -67,12 +69,12 @@ If you want to import your generated ZIP into Anypoint Studio, you need to packa
 > mvn clean package -DattachMuleSources
 ----
 
-The `-DattachMuleSources` argument tells the packager to add a mule-src folder inside the META-INF directory with an exact copy of your application structure at design time.
+The `-DattachMuleSources` argument tells the plugin to add a mule-src folder inside the META-INF directory with an exact copy of your application structure at design time.
```

Comment:
This property is only available on the 3.x version of the plugin. Need to remove this part.